README.md

```diff
@@ -25,36 +29,39 @@ Then include the `EditorConfig.pri` file in your .pro project file.
 include(editorconfig-core-qt/EditorConfig.pri)
 ```
 
-It can then be used within your Qt application.
+### CMake
+
+The easiest way should be to use CPM (https://github.com/cpm-cmake/CPM.cmake). Then,
+inside your CMake project do:
+
+```CMake
+# Or any other sha1 you want.
+CPMAddPackage("gh:editorconfig/editorconfig-core-qt#master")
+```
+
+You can alternatively add it as a git submodule and manually use `add_subdirectory()` to add the `CMakeLists.txt` file in your project.
+
+## Usage
+Once built, the library can then be used within your Qt application.
 
 ```c++
 #include <EditorConfig>
 
+...
+
 EditorConfigSettings settings = EditorConfig::getFileSettings("path/to/myfile.txt");
 for(auto setting : settings.toStdMap()) {
     std::cout << qUtf8Printable(setting.first) << "=" << qUtf8Printable(setting.second) << std::endl;
 }
 ```
 
-## How to use EditorConfig Qt5 Core (CMake)
-
-The easiest way, should be to use CPM (https://github.com/cpm-cmake/CPM.cmake). Then,
-inside your application do:
-
-``` CMake
-# Or any other sha1 you want.
-CPMAddPackage("gh:editorconfig/editorconfig-core-qt#master")
-```
-
-The follow instructions for QMake for code integration.
-
 ## Development
 
-This has been developed with Qt 5.14, however previous versions *may* work. This has only been tested on Windows.
+This has been developed with Qt 5.15+ however previous versions *may* work. To develop/test this module you can use the following instructions:
 
-1. Open `src/editorconfig-app/editorconfig-app.pro` with Qt Creator
-1. Build the release version (this will place the exe in a `build` directory in the root of this source code repository)
-1. Using (cmd|powershell), navigate to the `build` directory
-1. Execute `cmake ..`
-1. Execute `ctest -C Release` (no clue why it needs the `-C Release`)
+1. Open `CMakeLists.txt` with Qt Creator
+1. Configure the project and set `BUILD_EDITORCONFIG_CMD` to `ON`
+1. Build the project
+1. Using (cmd|powershell), navigate to the output directory
+1. Execute `ctest`
```

Comment:
https://doc.qt.io/qtcreator/creator-autotest.html

Reply:
I couldn't get `ctest` to work at all within Qt Creator. It never shows that there are any tests to run. Maybe I'm doing something wrong.